pkg/operator/ceph/object/rgw.go

```diff
@@ -71,6 +71,12 @@ func (c *clusterConfig) createOrUpdateStore() error {
 		return errors.Wrap(err, "failed to start rgw pods")
 	}
 
+	objContext := NewContext(c.context, c.store.Name, c.store.Namespace)
+	err := enableRGWDashboard(objContext)
+	if err != nil {
+		logger.Warningf("Dashboard disabling failed for rgw %v", err)
```

Comment:
Typo: disable->enable.
Rook uses the error message pattern "failed to..."
Separate error from additional message by a period (`.`)

```suggestion
		logger.Warningf("failed to enable dashboard for rgw. %v", err)
```